projects/engine-vscode/src/webview.ts

```diff
@@ -1,14 +1,18 @@
-import { PluginConnector, Profile, Message, PluginConnectorOptions, ExternalProfile } from "@remixproject/engine"
+import { PluginConnector, PluginConnectorOptions } from "@remixproject/engine"
+import { Message } from '../../utils/src/types/message'
+import { Profile, ExternalProfile } from '../../utils/src/types/profile'
+
 import { ExtensionContext, ViewColumn, Webview, WebviewPanel, window, Uri } from 'vscode'
 import { join, parse as parsePath } from 'path'
-import { promises as fs } from 'fs'
+import { promises as fs, watch } from 'fs'
 import { get } from 'https'
 import { parse as parseUrl } from 'url'
 
 interface WebviewOptions extends PluginConnectorOptions {
   /** Extension Path */
   context: ExtensionContext
   column?: ViewColumn
```

Comment:
although all the other properties are self explanatory, could you add a comment for that one?

Reply:
Sure. For a vscode developer it's quite obvious. When you divide your editor in several column it's the index in which you want to add your plugin. A little bit like the location in remix-ide